src/main/java/de/doubleslash/keeptime/view/ProjectsListViewController.java

```diff
@@ -151,13 +157,26 @@ public ProjectsListViewController(final Model model, final Controller controller
       searchTextField.setPromptText("Search");
    }
 
+   private void addProjectToProjectSelectionNodeMap(final Project project) {
+      if (project.isEnabled()) {
+         final Pane projectElement = addProjectToProjectList(project);
+         final Label projectNameLabel = (Label) projectElement.getChildren().get(0);
+         projectNameLabel.setTooltip(new Tooltip(projectNameLabel.getText()));
```

Comment:
I meant to put the tooltip logic into the "addProjectToProjectList" method, as this creates the ui representation of the project (which should also include the tooltip).
When doing this, please rename the method also, as it currently does not do what it says.